backup/setup/roles.json

```diff
@@ -127,6 +127,9 @@
 			"ADMIN_VIEW",
 			"HELPDESK_EDIT",
 			"HELPDESK_VIEW",
+			"NEWS_CREATE",
```

Comment:
administrators should extend teachers permission instead of users maybe, a teacher which is role id `0000d186816abba584714c95`  already has these roles.